umbraco/Umbraco.CMS.Backoffice
# Tiptap: Render toolbar with row and group configuration

## Description

Renders the Tiptap toolbar configuration with rows and groups.

**Configuration**
![Screenshot 2024-09-30 120124](https://github.com/user-attachments/assets/ebdd5b13-96dd-4366-8f91-8bd84efe4060)

**Rendered toolbar**
![Screenshot 2024-09-30 120137](https://github.com/user-attachments/assets/e1505aef-b00e-4009-9b1e-111cefd7a1b0)


## Types of changes


## How to test?

In the Tiptap data-type toolbar configuration, add buttons to groups and rows. In the property-editor, see that the toolbar buttons are rendered correctly.
